### PR TITLE
Add scope for last ten transactions on home page

### PR DESCRIPTION
**Pull request by nickzhihovich**
- Add scope for last ten transactions on home page

### CODE REVIEW

File: app/models/transaction.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@ class Transaction < ApplicationRecord
 
   scope :balance_transactions, -> { where(transactinable_type: 'BalanceTransaction') }
   scope :category_transactions, -> { where(transactinable_type: 'CategoryTransaction') }
+  scope :last_ten, -> { includes(:transactinable).limit(10).order('id desc') }
```
Comment (sinpai) [inline app/models/transaction.rb]: Isn't it better to use `order(id: :desc)` instead of string?

**Review by sinpai — COMMENTED** [review]
Suggestion

File: app/models/transaction.rb (hunk 1 above)
Comment (marderer) [reply app/models/transaction.rb]: @sinpai `order(id: :desc)` only sorts, I want take 10 records

Comment (AleksSenkou) [reply app/models/transaction.rb]: @marderer can't you go with:
```ruby
includes(:transactinable).order('id desc').limit(10)
```
?

File: app/models/user.rb [hunk 2]
```diff
@@ -21,5 +21,7 @@ def free_balance
 
   delegate :balance_transactions, to: :transactions
   delegate :category_transactions, to: :transactions
+  delegate :last_ten, to: :transactions
```
Comment (AleksSenkou) [inline app/models/user.rb]: @marderer why have you delegated it and hadn't used?

**Review by AleksSenkou — APPROVED** [review]
LGTM

**Pull request merged**